src/com/team2502/robot2018/subsystem/ElevatorSubsystem.java

```diff
@@ -49,6 +49,7 @@ public ElevatorSubsystem()
         pidTuner = new SendablePIDTuner(this, this);
 
         DashboardData.addUpdater(this);
+        calibrateEncoder();
```

Comment:
tell drive team that the elevator must be down when code is deployed / robot is turned on